src/2d/colate2.f

```diff
@@ -171,24 +172,33 @@ subroutine colate2 (badpts, len, lcheck, nUniquePts, lbase)
 c
 c     sorting uses one dimensional packing of 2D indices
 c     check if domain will fit  in integer*4.
-c     if not, just leave the duplicate, but rememer that efficiency 
+c     if not, just leave the duplicate, but rememer that efficiency
 c     of grids won't be correct (since divide by number of flaged pts in grid)
 c     If necessary, do whole process in integer*8 - then will have enough
 c     room, but will have to convert quicksort routine and drivesort
 c     the variable largestIntEquiv already declared integer*8 above.
-      largestIntEquiv =  iregsz(lcheck)+mbuff + 
+      ifac1 = iregsz(lcheck)
+      ifac2 = jregsz(lcheck)
+      largestIntEquiv =  ifac1+mbuff +
+     .             (ifac1+2*mbuff)*(ifac2+mbuff)
+      write(*,*) 'level ',lcheck
```

Comment:
These print statements kind of look like they always get printed.  Should these be commented out?